docs/faas/faas-metrics.md

```diff
@@ -37,21 +37,21 @@ type and units.
 Below is a table of FaaS invocation metric instruments.
 
 | Name                   | Instrument Type ([*](/docs/general/metrics-general.md#instrument-types)) | Unit         | Unit ([UCUM](/docs/general/metrics-general.md#instrument-units)) | Description                                                                  |
-|------------------------|---------------------------------------------------|--------------|-------------------------------------------|------------------------------------------------------------------------------|
-| `faas.invoke_duration` | Histogram                                         | milliseconds | `ms`                                      | Measures the duration of the invocation                                      |
-| `faas.init_duration`   | Histogram                                         | milliseconds | `ms`                                      | Measures the duration of the function's initialization, such as a cold start |
```

Comment:
Shouldn't these changes come with explicit bucket boundary advice so that the distribution is useful out of the box?

This question applies to all the histogram metrics that appear in this PR.